### PR TITLE
Some type fixing after interfaces refactor

### PR DESCRIPTION
**Pull request by chrichri17**
Depends on #75. Just some frontend fixing

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #73** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/73" target="_blank">Graphite</a> 
    * **PR #74** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/74" target="_blank">Graphite</a> 
      * **PR #77** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/77" target="_blank">Graphite</a>  👈
        * **PR #78** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/78" target="_blank">Graphite</a> 
          * **PR #79** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/79" target="_blank">Graphite</a> 
      * **PR #75** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/75" target="_blank">Graphite</a> 
        * **PR #76** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/76" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/77?utm_source=stack-comment).

File: frontend/src/views/Quiz.vue [hunk 1]
```diff
@@ -36,7 +33,7 @@ onBeforeMount(() => {
                 );
 
                 for (const question of response.data.value?.questions || []) {
-                    useFetch<QuizQuestion>(
+                    useFetch<Quizoot.Question>(
```
Comment (chrichri17) [inline frontend/src/views/Quiz.vue]: this is why I have `ApiResponse = any` right now. I'll think about a fix

Comment (YacineSteeve) [reply frontend/src/views/Quiz.vue]: I understand

**Review by YacineSteeve — APPROVED** [review]
💯

**Pull request merged**